lib/SearchResultsList/SearchResultsList.js

```diff
@@ -267,9 +269,10 @@ const SearchResultsList = ({
 SearchResultsList.propTypes = propTypes;
 
 SearchResultsList.defaultProps = {
+  columnWidths: null,
```

Comment:
Let's use `defaultColumnWidths` as default prop value, and we can remove `columnWidths || defaultColumnWidths` above